### ISSUE TITLE
Ubuntu PPA Key Expired

### ISSUE THREAD

**Issue by zac-hopkinson** [bazelbuild/bazel#11470]
### Description of the problem / feature request:

The Ubuntu PPA key has expired which is causing installation of Bazel to fail.

### Feature requests: what underlying problem are you trying to solve with this feature?

Trying to install Bazel on Ubuntu using the instructions here: https://docs.bazel.build/versions/master/install-ubuntu.html

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

```bash
docker run --rm -it ubuntu:18.04 bash
apt-get update && apt-get install -y gnupg curl
curl https://bazel.build/bazel-release.pub.gpg | gpg -v
exit
```

Outputs:

```plain
gpg: Note: signature key 3D5919B448457EE0 expired Sat May 23 13:10:53 2020 UTC
pub   rsa4096 2016-05-24 [SC] [expired: 2020-05-23]
      71A1D0EFCFEB6281FD0437C93D5919B448457EE0
uid           Bazel Developer (Bazel APT repository key) <bazel-dev@googlegroups.com>
sig        DD3EF963991F1EC2 2016-10-28   [User ID not found]
sig        3D5919B448457EE0 2018-05-24   [selfsig]
sig        3D5919B448457EE0 2016-05-24   [selfsig]
sub   rsa4096 2016-05-24 [E] [expired: 2020-05-23]
sig        3D5919B448457EE0 2018-05-24   [keybind]
```

### What operating system are you running Bazel on?

Ubuntu 18.04

### What's the output of `bazel info release`?

N/A

###  Have you found anything relevant by searching the web?

Related issue the last time this happened: https://github.com/bazelbuild/bazel/issues/5261

**Comment by LuanBaviloni** [comment]
Any news on this?

**Comment by philwo** [comment]
Sorry 😬 I'm currently fixing this, will ping here when the new keys have been uploaded.

**Comment by philwo** [comment]
I think I did everything that's required to fix this:
[x] Extend validity of Bazel's release signing key until May 2022
[x] Push updated key to https://storage.googleapis.com/bazel-apt/doc/apt-key.pub.gpg
[x] Push updated key to https://bazel.build/bazel-release.pub.gpg via https://github.com/bazelbuild/bazel-website/pull/247.
[x] Push updated key to http://pool.sks-keyservers.net/
[x] Push updated key to https://keyserver.ubuntu.com/

Please note that it can take ~1 hour for bazel.build to give the new key due to the time it requires to build the website and for caches to expire.

**Comment by jin** [comment]
You may have to manually update your apt keyring with the new key:

```
$ curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
$ sudo apt-get update
```

**Comment by malkia** [comment]
> You may have to manually update your apt keyring with the new key:
> 
> ```
> $ curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
> $ sudo apt-get update
> ```

This works for me now (Earlier this week, these same commands did nothing for me).

**Comment by gslndlb** [comment]
> You may have to manually update your apt keyring with the new key:
> 
>     $ curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
>     $ sudo apt-get update

Fix tested and it worked for me.

**Comment by tomkukral** [comment]
I'm still getting expired key

```
root@bf62d2ee246a:/# curl https://bazel.build/bazel-release.pub.gpg | gpg -v
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3199  100  3199    0     0   7124      0 --:--:-- --:--:-- --:--:--  7124
gpg: Note: signature key 3D5919B448457EE0 expired Tue May 24 06:37:14 2022 UTC
pub   rsa4096 2016-05-24 [SC] [expired: 2022-05-24]
      71A1D0EFCFEB6281FD0437C93D5919B448457EE0
uid           Bazel Developer (Bazel APT repository key) <bazel-dev@googlegroups.com>
sig        3D5919B448457EE0 2020-05-24   [selfsig]
sub   rsa4096 2016-05-24 [E] [expired: 2022-05-24]
sig        3D5919B448457EE0 2020-05-24   [keybind]
```

Is there any CDN for `bazel.build`? Can you invalidate the cache or share new gpg key here?

**Comment by mzubala** [comment]
Looks like the key expired again yesterday (2022-05-24).

**Comment by sepi83** [comment]
The key does not work

root@015a79c5c628:~/tensorflow# sudo apt update && sudo apt install bazel-5.2
Get:1 https://storage.googleapis.com/bazel-apt stable InRelease [2256 B]
Err:1 https://storage.googleapis.com/bazel-apt stable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 3D5919B448457EE0
Hit:2 http://ports.ubuntu.com/ubuntu-ports jammy InRelease
Hit:3 http://ports.ubuntu.com/ubuntu-ports jammy-updates InRelease
Hit:4 http://ports.ubuntu.com/ubuntu-ports jammy-backports InRelease
Hit:5 http://ports.ubuntu.com/ubuntu-ports jammy-security InRelease
Reading package lists... Done
W: GPG error: https://storage.googleapis.com/bazel-apt stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 3D5919B448457EE0
E: The repository 'https://storage.googleapis.com/bazel-apt stable InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@015a79c5c628:~/tensorflow# # # #

**Comment by RCOONa** [comment]
![image](https://user-images.githubusercontent.com/115844647/209828261-d55a74da-65f7-4b7f-bfe8-7919684ff8f8.png)
![image](https://user-images.githubusercontent.com/115844647/209828436-6ea1495a-5642-4562-a9e3-31f8a07b8ee7.png)
![image](https://user-images.githubusercontent.com/115844647/209828732-ca207ef5-0e72-4d2a-977f-cbdf68437013.png)

Oh... no..... today is... 2022-12-28(Wen)
